.github/workflows/ci.yml

```
@@ -0,0 +1,120 @@
+            echo "lock_file_exists=true" >> $GITHUB_OUTPUT
+          else
+            echo "lock_file_exists=false" >> $GITHUB_OUTPUT
+          fi
+
+      - name: Setup Node.js with cache
+        if: steps.check-lock-file.outputs.lock_file_exists == 'true'
+        uses: actions/setup-node@v4
+        with:
+          node-version: '20'
+          cache: 'npm'
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)